plugins/release/skills/release-migrate-variant-periodics/README.md

```diff
@@ -0,0 +1,350 @@
+```
+
+### 5. Branch Metadata
+```yaml
+# Before (4.17)
+zz_generated_metadata:
+  branch: release-4.17
+  org: openshift
+  repo: cloud-credential-operator
+
+# After (4.18)
+zz_generated_metadata:
+  branch: release-4.18
+  org: openshift
+  repo: cloud-credential-operator
+```
+
+### 6. Cron Schedules
+```yaml
+# Before (randomized for 4.17)
+cron: 11 4 14 8 *
+
+# After (re-randomized for 4.18)
+cron: 37 15 14 8 *
+```
+
+**Note:** Cron schedules are regenerated to avoid thundering herd problems. The new schedules maintain the same frequency (monthly, weekly, daily) but use different randomized times.
+
+## Resource Specification Format
+
+The skill accepts version numbers in several formats:
+
+**Input formats:**
+- `4.17` - Simple major.minor format (preferred)
+- `release-4.17` - Full branch name format
+- `4.17.0` - With patch version (will be normalized to 4.17)
+
+**File pattern:**
+- Source: `*-release-{from-version}__periodics.yaml`
+- Target: `*-release-{to-version}__periodics.yaml`
+
+**Path specification:**
+- Default: `ci-operator/config/` (searches all subdirectories)
+- Specific repo: `ci-operator/config/openshift/etcd`
+- Organization: `ci-operator/config/openshift-priv`
+
+## Workflow
+
+1. **Branch Verification (Safety Check)**
+   - Check current git branch
+   - Display branch name to user
+   - Warn if on main/master branch
+   - Ask user to confirm proceeding with modifications
+   - Exit if user declines
+
+2. **Input Validation**
+   - Normalize version numbers to `{major}.{minor}` format
+   - Validate version format and relationship
+   - Determine search path
+
+3. **File Discovery**
+   - Search for files matching pattern
+   - Display list of files found
+   - Get user confirmation
+
+4. **Conflict Resolution**
+   - Check if target files already exist
+   - Ask user whether to overwrite
+   - Build migration plan
+
+5. **Transformation**
+   - Read source file
+   - Transform version references
+   - Regenerate cron schedules
+   - Validate YAML syntax
+
+6. **File Creation**
+   - Write new periodic files
+   - Track success/failure
+   - Handle errors gracefully
+
+7. **Reporting**
+   - Display migration summary
+   - Report successes and failures
+   - Suggest next steps
+
+## Usage Examples
+
+### Migrate all periodics from 4.17 to 4.18
+```
+/release:migrate-periodics 4.17 4.18
+```
+Searches all of `ci-operator/config/` for files matching `*-release-4.17__periodics.yaml`.
+
+### Migrate periodics for a specific repository
+```
+/release:migrate-periodics 4.18 4.19 ci-operator/config/openshift/cloud-credential-operator
+```
+Limits search to the cloud-credential-operator directory.
+
+### Migrate all openshift organization periodics
+```
+/release:migrate-periodics release-4.19 release-4.20 ci-operator/config/openshift
+```
+Processes all repositories under the openshift organization.
+
+### Migrate openshift-priv periodics
+```
+/release:migrate-periodics 4.17 4.18 ci-operator/config/openshift-priv
```

Comment:
AFAIK openshift-priv content is entirely automatically managed, I would not encourage agents to touch anything there.